src/screens/profile-screen/ProfileScreen.tsx

```diff
@@ -1,77 +1,63 @@
-import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import { makeGetProfile } from 'audius-client/src/pages/profile-page/store/selectors'
 import { Dimensions, Text, View } from 'react-native'
 
-import IconAlbum from 'app/assets/images/iconAlbum.svg'
-import IconCollectibles from 'app/assets/images/iconCollectibles.svg'
-import IconNote from 'app/assets/images/iconNote.svg'
-import IconPlaylists from 'app/assets/images/iconPlaylists.svg'
-import IconRepost from 'app/assets/images/iconRepost.svg'
-import TopTabNavigator from 'app/components/app-navigator/TopTabNavigator'
-import { BaseStackParamList } from 'app/components/app-navigator/types'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { makeStyles } from 'app/styles/makeStyles'
 
-// We might need to allow BaseStackParamList to be generic here
-// to get all the relevant params
-type Props = NativeStackScreenProps<BaseStackParamList, 'track'>
+import { CoverPhoto } from './CoverPhoto'
+import { ExpandableBio } from './ExpandableBio'
+import { ProfileMetrics } from './Metrics'
+import { ProfilePhoto } from './ProfilePhoto'
+import { ProfileTabNavigator } from './ProfileTabNavigator'
+import { ProfileSocials } from './Socials'
+
+const getProfile = makeGetProfile()
 
 const screenHeight = Dimensions.get('window').height
 
-const TracksTab = () => {
-  return <Text>Tracks Tab</Text>
-}
-const RepostsTab = () => {
-  return <Text>Reposts Tab</Text>
-}
-const AlbumsTab = () => {
-  return <Text>Albums Tab</Text>
-}
-const PlaylistsTab = () => {
-  return <Text>Playlists Tab</Text>
-}
-const CollectiblesTab = () => {
-  return <Text>Collectibles Tab</Text>
-}
+const useStyles = makeStyles(({ palette, typography, spacing }) => ({
+  screen: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: screenHeight
+  },
+  header: {
+    backgroundColor: palette.white,
+    paddingTop: spacing(8),
+    paddingHorizontal: spacing(3)
+  },
+  username: {
+    ...typography.h1,
+    color: palette.neutral
+  },
+  handle: {
+    ...typography.h2,
+    color: palette.neutralLight4,
+    marginBottom: spacing(4)
+  }
+}))
+
+const ProfileScreen = () => {
+  const styles = useStyles()
+  const { profile } = useSelectorWeb(getProfile)
+
+  if (!profile) return null
 
-const ProfileScreen = ({ navigation }: Props) => {
   return (
-    <View
-      style={{
-        display: 'flex',
-        flexDirection: 'column',
-        height: screenHeight
-      }}
-    >
-      <Text style={{ flex: 1 }}>Example profile screen</Text>
+    <View style={styles.screen}>
+      <CoverPhoto profile={profile} />
+      <ProfilePhoto profile={profile} />
+      <View style={styles.header}>
+        <Text accessibilityRole='header' style={styles.username}>
```

Comment:
Oh cool, I didn't realize there was an `accessibilityRole`

Reply:
yeah! it all seems quite a bit less powerful than web, but at least there is something